gulp/tasks/test_ci.js

```diff
@@ -0,0 +1,14 @@
+(function () {
+    'use strict';
+
+    var gulp = require('gulp'),
+        karma = require('karma').server,
+        path = require('path'),
+        configFile = 'karma.ci.conf.js';
+
+    gulp.task('test_ci', function (callback) {
```

Comment:
I like the idea of running tests on multiple browsers, but it seems unnecessary to take the time to test all of them locally on every run. Most bugs will probably be caught by a single run of the test suite, but having extra defense in the form of cross-browser tests seems nice to have.